### ISSUE TITLE
Programmatically trigger data table row collapse

### ISSUE THREAD

**Issue by RockoTheeHut** [ironmansoftware/powershell-universal#3949]
### Summary of the new feature / enhancement

I want to be sale to trigger a row expand, or even an expand as well, for a data table row. Not being same to trigger a row collapse leads to confusing behavior of you refresh a data table. The next route I’m there indeed is displayed while the row status expanded. It’s very confusing.

i asked in the forms and got no reply.. https://forums.ironmansoftware.com/t/manually-collapse-a-table-row-onrowexpand-section/11580

it will be some time before we can upgrade to 5 so if possible it would be nice to have this in v4

### Proposed technical implementation details (optional)

An accessible element to set true or false for a data table row
